setup.cfg

```diff
@@ -36,6 +36,9 @@ devel =
 httpsupport =
     requests
     requests_toolbelt
+remotefs =
+    fsspec
+    requests
```

Comment:
I discovered in #223 that a dependency missing on my system was ``aiohttp`` in addition to those listed here

Reply:
When trying S3 URLs, I was missing the s3fs dependency. However, the reporting for this was very nice and user-friendly:

```
datalad -c 'datalad.url-handler.(^|.*::)s3://openneuro.org.class=datalad_next.url_operations.fsspec.FsspecUrlOperations' -c 'datalad.url-handler.(^|.*::)s3://openneuro.org.kwargs={"fs_kwargs": {"s3": {"anon": true}}}' download 's3://openneuro.org/ds004393/sub-26/func/sub-26_task-ImAcq_acq-task_run-05_bold.nii.gz demo.nii.gz'

download(error): demo.nii.gz [download failure] [Install s3fs to access S3]
```

This error comes from fsspec directly: https://github.com/fsspec/filesystem_spec/blob/e180aa859ef081215882b2d1b67d4bc33c040330/fsspec/registry.py#L114.

Interestingly, this mapping of protocols to dependencies and errors also exists for HTTP (https://github.com/fsspec/filesystem_spec/blob/e180aa859ef081215882b2d1b67d4bc33c040330/fsspec/registry.py#L70), and I did see it in #223 - but only in the special remote debug output, not bubbled up. It would be nice to find out what makes the s3 code handle this error better, and mirror it in the generic fsspec code.